libs/features/mas/web-components/src/merch-card.js

```diff
@@ -432,7 +432,9 @@ export class MerchCard extends LitElement {
                     >${this.actionMenuContent}</slot
                 >
                 <slot name="heading-xs"></slot>
-                <slot name="heading-m"></slot>
+                <slot name="heading-m">
+                    <slot name="payment-details"></slot>
```

Comment:
A `slot` inside another `slot` doesn't seem to be allowed as per chatgpt but the specs do not mention it.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/slot

technically this should be equal to:


```html
<slot name="payment-details"></slot>
<slot name="heading-m"></slot>
...
</slot>
``` 

what is the reason for nesting?

Reply:
@yesil 
The merch card body has a flex gap between all elements (see screenshot).
If I place the `payment-details` slot outside the `heading-m` slot, there will be a gap between the two elements and it will not look as on the XD.

![Screenshot 2024-08-13 at 10 52 54](https://github.com/user-attachments/assets/31c3c1b3-05c9-4fb0-9509-787db40d0468)

Reply:
@mirafedas in this case, why it can't be a part of `heading-m` ?

Reply:
@yesil we need a way to distinguish which text goes into `heading-m` and which into `payment-details` because both texts should be on different lines and have different styles

Reply:
@mirafedas we need to find a way to do it without nested slots though.

Reply:
I replaced the slot with a div, now should be good 👍